### DISCUSSION TITLE
Initial Copy of eBook Files into Calibre (employing DeDRM)

### DISCUSSION THREAD

**Discussion by COHikerGirl** [noDRM/DeDRM_tools#164] [Q&A]
I am importing Kindle eBooks into Calibre, which has been configured to use the DeDRM plugin.

Calibre supports the input formats: CBZ, CBR, CBC, CHM, EPUB, FB2, HTML, LIT, LRF, MOBI, ODT, PDF, PRC, PDB, PML, RB, RTF, SNB, TCR, TXT. 

Its output formats include file types: EPUB, FB2, OEB, LIT, LRF, MOBI, PDB, PML, RB, PDF, SNB, TCR, TXT.

But the set of Kindle files copied directly from my deceased Dad's Kindle database directory into a folder on my windows laptop (via USB cable) include a whole SET of filetypes associated with each book.  Variously, these filetypes include the extensions:
.apnx, .azw, .azw3, .ea., .han, .mbp, 
.mbpv2, .phl, .sa, .upd 

When first populating a Calibre library via the 'Add Books' button, which of these Kindle files should be selected to add?  I am presuming I should add JUST the .azw or .azw3 eBook files.  But I wanted to double-check with y'all whether I should be loading any of the others together with them?  I am presuming NOT.

I've learned what several of these additional files are, and it seems to me that I should NOT include them among 'eBook files" that can/should be added to a Calibre library:

- The .mbp and .mbpV2 files are Mobipocket Notes Files, which contain the e-book notes the user created in an e-book (such as corrections, annotations, etc.).

- The .han files contain Kindle information about an eBook such as notes, bookmarks, and reading progress, which are similar to .MBP files but replaced them. 

- The .apnx files are Kindle page number index files used to mark pages in Kindle eBooks.  

- The .ea files are Kindle "end action" formats used to The information in the EA file is used to display the "More by this author" and "Customers who bought this book also bought" lists that appear after reading the last page of a Kindle book.  

- The .phl files demark Kindle's "Popular Highlights" information on sections or phrases within the eBook that have been highlighted by Kindle users.

MY QUESTION:  Am I correct that ALL of these file types (apart from .azw and .azw3) should be skipped and NOT imported into the Calibre library -- that they would NOT contain any data that Calibre would handle and include in its import?

**Comment by ElleKayEm** [comment]
Yes, you just want the main book file.  When you click the add books button and the select books dialog box opens, there's a drop down on the lower right that will let you filter file types.  One of the choices is Kindle books.

These are copied directly from the Kindle?  If you're seeing any KFX format, those will have to be handled differently.
